CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+How to contribute to Cozy MyAccounts?
+====================================
+
+Thank you for your interest in contributing to Cozy! There are many ways to contribute, and we appreciate all of them.
+
+
+Security Issues
+---------------
+
+If you discover a security issue, please bring it to our attention right away! Please **DO NOT** file a public issue, instead send your report privately to security AT cozycloud DOT cc.
+
+Security reports are greatly appreciated and we will publicly thank you for it. We currently do not offer a paid security bounty program, but are not ruling it out in the future.
+
+
+Bug Reports
+-----------
+
+While bugs are unfortunate, they're a reality in software. We can't fix what we don't know about, so please report liberally. If you're not sure if something is a bug or not, feel free to file a bug anyway.
+
+Opening an issue is as easy as following [this link][issues] and filling out the fields. Here are some things you can write about your bug:
+
+- A short summary
+- What did you try, step by step?
+- What did you expect?
+- What did happen instead?
+- What is the version of the Cozy MyAccounts?
```

Comment:
What about putting an issue template in the repo?

From my experience, it can be useful when non-tech users are reporting issues and to have easily processable issues.